### ISSUE TITLE
### <span aria-hidden="true">❌</span> Deploy Preview for *digital-crystal-protocol* failed.

### ISSUE THREAD

**Issue by FatherTimeSDKP** [FatherTimeSDKP/FatherTimeSDKP-#7]
### <span aria-hidden="true">❌</span> Deploy Preview for *digital-crystal-protocol* failed.

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | e1f947561e9421d070762fb4a77878260ab82e38 |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/projects/digital-crystal-protocol/deploys/6a2719636c09f300089c4a7e |

_Originally posted by @netlify[bot] in https://github.com/FatherTimeSDKP/FatherTimeSDKP/pull/58#issuecomment-4652863914_
